utils/swift_build_support/tests/products/test_llvm.py

```diff
@@ -2,7 +2,7 @@
 #
 # This source file is part of the LLVM.org open source project
 #
-# Copyright (c) 2014 - 2017 Apple Inc. and the LLVM project authors
```

Comment:
I noticed that this copyright notice says "and the LLVM project authors", instead of Swift. It's the only file in the codebase like this. Was that purposeful @gottesmm? It looks like you added this in

Reply:
It was a mistake.

Reply:
It looks like I put in the llvm header instead of the swift header.

Reply:
I'll change it